spec/controllers/confirmations_controller_spec.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
```

Comment:
remove it

Reply:
@kth-wtag apu, all these lines were removed in #1 when resolving comments.

Reply:
@kth-wtag apu, done.